package.json

```diff
@@ -27,7 +27,7 @@
     "async": "^2.6.1",
     "ethashjs": "~0.0.7",
     "ethereumjs-block": "~2.1.0",
-    "ethereumjs-common": "~0.6.0",
+    "ethereumjs-common": "~1.0.0",
```

Comment:
We should change this to a caret to allow for new minor versions (should be safe)<sup>[\[1\]](https://stackoverflow.com/a/22345808/1267663)</sup>

```suggestion
    "ethereumjs-common": "^1.0.0",
```

Reply:
Thanks. Fixed.